### ISSUE TITLE
Does not work well with ExtractText Plugin

### ISSUE THREAD

**Issue by nadavsinai** [BohdanTkachenko/webpack-split-by-path#10]
I'm trying to use your awesome plugin together with ExtractText plugin, in begining I got problems with text.forEach is not a function, but thanks to the aid of @greenGremlin I found that adding ignore regexps containing the css-loader path and /.css$/ (and the same for less in my case ) solves the error, however the created css does not contain all of the code from the vendor bucket (again relevant to my case)
this issues is [related to this issue](https://github.com/webpack/extract-text-webpack-plugin/issues/92)
webpack/extract-text-webpack-plugin/#92

the relevant parts of my config shortened for brevity, actually using 3 files to merge into webpack config so I have reference of the created plugin in the loaders section:

``` javascript
plugins:[new SplitByPathPlugin([{
        name: 'vendor',
        path: [path.join(basePath, 'node_modules'), path.join(basePath, 'frontEnd', 'lib'), path.join(basePath, 'frontEnd', 'core', 'libs')]
    }], {ignore: [/style-loader/, /css-loader/, /less-loader/, /\.less$/, /\.css$/]}),new ExtractTextPlugin('css/vendor.css', {
        allChunks: false
    }), new ExtractTextPlugin('css/[name].css', {
        allChunks: false
    })],
module:{loaders:[
{test: /\.css(\?.*)?$/, loader: plugins.extractCss.extract('style-loader', "css-loader?sourceMap", {allChunks: false, disable: false})},
{
        test: /\.less(\?.*)?$/,
        loader: plugins.extractLess.extract('style-loader', 'css-loader?sourceMap!less-loader?{"sourceMap":true,"modifyVars":' + JSON.stringify(colors) + '}', {
            allChunks: false,
            disable: false
        })
]]}


Do you have any idea how to overcome this?
thanks
```

**Comment by aldendaniels** [comment]
:+1: This is affecting me as well.

**Comment by aldendaniels** [comment]
@nadavsinai Looks like this has been fixed in master.

@BohdanTkachenko - Could you push the new version to NPM? Thanks!

**Comment by BohdanTkachenko** [comment]
Sorry for the delay. I published version 0.0.8. Closing this issue. Feel free to re-open it if issue still reproduces. Thanks!

**Comment by nadavsinai** [comment]
10x it works now!
